tutteli-gradle-publish/src/test/groovy/ch/tutteli/gradle/publish/PublishPluginIntTest.groovy

```diff
@@ -374,8 +475,8 @@ class PublishPluginIntTest {
             "<dependency>$NL_INDENT" +
             "<groupId>ch.tutteli.atrium</groupId>$NL_INDENT" +
             "<artifactId>atrium-cc-en_GB-robstoll</artifactId>$NL_INDENT" +
-            "<version>0.7.0</version>$NL_INDENT" +
-            "<scope>compile</scope>$NL_INDENT" +
+            "<version>$ATRIUM_VERSION</version>$NL_INDENT" +
+            "<scope>runtime</scope>$NL_INDENT" +
```

Comment:
Why did that change to runtime?

Reply:
I don't know, really. It is probably connected to using the implementation configuration.

However, it does not matter for consumers, does it? That is why I just accepted it.

Reply:
ah.. I overlooked the implementation. Can you quickly verify locally if it changes to compile if you use `api` instead of `implementation`. However, we can keep implementation, that's perfectly fine.

Reply:
It does indeed change to `compile` when using `api`. That makes a lot of sense, actually :)

Would you prefer having the `api` case in the test? I think it does not matter much, since we only test _that_ dependencies are added. Whether they are added correctly does not need to be tested since it is not done by our code.

Reply:
implementation is fine